doc/rest-api/Metrics-backend.md

```diff
@@ -81,54 +55,54 @@ Example response:
 
 Returns ```200 OK``` and two elements:
 
-* "hosts" containing a list of XMPP host names available on the server,
-* "metrics" containing a list of metrics available on the server.
+* "hosts" A list of XMPP host names available on the server.
+* "metrics" - A list of per-host metrics.
+* "global" A list of global metrics.
 
 ### GET /api/metrics/all
 
 Returns ```200 OK``` and an element:
 
-* "metrics" containing a list of summed metrics for all server hosts.
+* "metrics" - A list of aggregated (sum of all domains) per-host metrics with their values.
 
 ### GET /api/metrics/all/:metric
 
 On success returns ```200 OK``` and an element:
 
-* "metric" containing a metric :metric that summs up values for all server hosts.
+* "metric" - An aggregated (sum of all domains) per-host metric.
 
-Returns ```404 Not Found``` when metric :metric couldn't been found.
+Returns ```404 Not Found``` when metric `:metric` couldn't been found.
 
 ### GET /api/metrics/host/:host
 
 On success returns ```200 OK``` and an element:
 
-* "metrics" containing a list of metric values for host :host.
+* "metrics" - A list of per-host metrics and their values for host `:host`.
 
 Returns ```404 Not Found``` when host :host couldn't been found.
 
 ### GET /api/metrics/host/:host/:metric
 
 On success returns ```200 OK``` and an element:
 
-* "metric" containing a metric :metric value for host :host.
+* "metric" - A per-host metric `:metric` and its value for host `:host`.
 
 Returns ```404 Not Found``` when the pair (host :host, metric :metric) couldn't been found.
```

Comment:
the pair `(host :host, metric :metric)` does not exist (or "can't be found", if you prefer)